apps/web-giddh/src/app/reports/components/sales-register-component/sales.register.component.html

```diff
@@ -1,8 +1,39 @@
+
+<div class="relative">
+    <hamburger-menu [pageHeading]="''"></hamburger-menu>
+    <div class="report-breadcrum">
+        <ul class="list-head list-inline d-inline-block middle nav-report">
+            <li><a href="#">Report</a> <span class="d-inline-block pl-1"> ></span></li>
+            <li><a href="#">Sales Register</a> <span class="d-inline-block pl-1"> ></span></li>
+            <li>
+                <div class="select-style width-200 select-year">
+                    <select name="">
+                        <option value="">August 2017</option>
```

Comment:
Why static text is added?

Reply:
This is old code I have replaced only position